tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1296,101 +1288,80 @@ def test_continuous_reads_during_kpods_down(self):
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
                                  f"Logs which contain pass: {resp[1]}")
-        LOGGER.info("Step 7: Verified status for In-flight READs/Verify DI while %s (K) pods "
+        LOGGER.info("Step 4: Verified status for In-flight READs/Verify DI while %s (K) data pods "
                     "going down.", self.kvalue)
 
-        LOGGER.info("Step 8: Create multiple buckets and run IOs")
-        self.test_prefix = 'test-35773-1'
-        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix, nsamples=2,
-                                                    nclients=2)
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("STEP 5: Create IAM user and perform WRITEs-READs-Verify-DELETEs with "
```

Comment:
this test is about Read so do we want to do DELETEs?

Reply:
> this test is about Read so do we want to do DELETEs?

This is last step where we are performing IOs. as we discussed yesterday. It is not about READs WRITEs or DELETEs.